weldx/geometry.py

```diff
@@ -2377,6 +2410,102 @@ def spatial_data(
         )
         return SpatialData.from_geometry_raster(rasterization, closed_mesh)
 
+    def _shape_points_and_triangles(
+        self, shape_raster_data: np.array, offset: int, cw_ordering: bool
+    ):
+        if isinstance(self._profile, VariableProfile):
+            raise NotImplementedError
+
+        shape_raster_data = shape_raster_data.swapaxes(1, 2)
```

Comment:
I noticed in my 3D Viewer that the model orientation seems 'sideways', is this because of this line?